frontend/scripts/flutter_release_build/build_flowy.dart

```diff
@@ -0,0 +1,22 @@
+import 'dart:io';
+
+part 'tool.dart';
+
+Future<void> main(List<String> args) async {
```

Comment:
@a-wallen Are there any reasons for using Dart to write a script instead of using `Shell` or defining it in `desktop.toml`?

Reply:
All contributors will have dart installed. I did have a version of the script written in `bash`, but I decided to go with dart since  I could use OOP.